Source/ARSLineProgressHelpers.swift

```diff
@@ -259,6 +259,7 @@ func ars_animateCircles(_ outerCircle: CAShapeLayer, middleCircle: CAShapeLayer,
 		outerAnimation.toValue = ARS_CIRCLE_ROTATION_TO_VALUE
 		outerAnimation.duration = ars_config.circleRotationDurationOuter
 		outerAnimation.repeatCount = ARS_CIRCLE_ROTATION_REPEAT_COUNT
+        outerAnimation.isRemovedOnCompletion = false
```

Comment:
Thanks for your PR!
Does this really fix the issue?
Could you please add one tab of padding to match the other lines of code?